src/_includes/docs/add-to-app/ios-project/embed-frameworks.md

```diff
@@ -0,0 +1,117 @@
+Use this method in the following use cases:
+
+* Members of your team can't install the Flutter SDK and CocoaPods
+* You don't want to use CocoaPods as a dependency manager in existing iOS apps
+
+#### Limitations {:#method-b-limits}
+
+{% render docs/add-to-app/ios-project/limits-common-deps.md %}
+
+#### Example project structure {:#method-b-structure}
+
+The following example assumes that you want to generate the
+frameworks to `/path/to/MyApp/Flutter/`.
+
+```console
+$ flutter build ios-framework --output=/path/to/MyApp/Flutter/
+```
+
+The resulting project structure should resemble this directory tree.
+
+```plaintext
+/path/to/MyApp/
+└── Flutter/
+    ├── Debug/
+    │   ├── Flutter.xcframework
+    │   ├── App.xcframework
+    │   ├── FlutterPluginRegistrant.xcframework (only if you have plugins with iOS platform code)
+    │   └── example_plugin.xcframework (each plugin is a separate framework)
+    ├── Profile/
+    │   ├── Flutter.xcframework
+    │   ├── App.xcframework
+    │   ├── FlutterPluginRegistrant.xcframework
+    │   └── example_plugin.xcframework
+    └── Release/
+        ├── Flutter.xcframework
+        ├── App.xcframework
+        ├── FlutterPluginRegistrant.xcframework
+        └── example_plugin.xcframework
+```
+
+:::warning
+Always put `Flutter.xcframework` and `App.xcframework` from the same directory.
+Mixing `.xcframework` imports from different directories
+(like `Profile/Flutter.xcframework` with `Debug/App.xcframework`)
+causes runtime crashes.
+:::
+
+#### Procedures
+
+You can either link or embed the generated frameworks
+into your existing app in Xcode.
+Use the method that is best for your project.
+
+##### Link on the frameworks
+
+Continuing with this example,
+you can drag the frameworks from
+the `/path/to/MyApp/Flutter/Release/` directory in the Finder
+to your target's
+**Build Settings > Build Phases > Link Binary With Libraries** settings.
+
+You must also open `MyApp.xcodeproj/project.pbxproj` (from Finder)
+and replace `path = Flutter/Release/example.xcframework;`
+with `path = "Flutter/$(CONFIGURATION)/example.xcframework";`
+for all added frameworks. The double quotes (`"`) are necessary.
+
+In the target's build settings, add `$(PROJECT_DIR)/Flutter/Release/`
+to the **Framework Search Paths** (`FRAMEWORK_SEARCH_PATHS`).
+
+{% render docs/captioned-image.liquid, image:"development/add-to-app/ios/project-setup/framework-search-paths.png", caption:"Update Framework Search Paths in Xcode" %}
+
+:::tip
+To use the simulator, embed the Debug version of the Flutter frameworks
+in your Debug build configuration.
+To do this, use `$(PROJECT_DIR)/Flutter/$(CONFIGURATION)`
+in the **Framework Search Paths** (`FRAMEWORK_SEARCH_PATHS`) build setting.
+This embeds the Release frameworks in the Release configuration,
+and the Debug frameworks in the Debug Configuration.
+:::
+
+##### Embed the frameworks
+
+To load the generated dynamic frameworks at runtime,
```

Comment:
Ok now I really have no idea what I'm talking about here, but I _think_ this wording isn't quite right.

My understanding is that embedding the framework changes how the framework is distributed by dumping the framework into the app's bundle. I _believe_ that embedding the framework doesn't change how the app loads the framework.

Consider something like:

```suggestion
Embed the dynamic frameworks into your app.
```